_health-care/_js/_components/veteran_name.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import _ from 'lodash';
+
+class VeteranName extends React.Component {
+  constructor() {
+    super();
+    this.state = {first: '', middle: '', last: '', suffix: ''};
+    this.handleFirstChange = this.handleFirstChange.bind(this);
+    this.handleMiddleChange = this.handleMiddleChange.bind(this);
+    this.handleLastChange = this.handleLastChange.bind(this);
+    this.handleSuffixChange = this.handleSuffixChange.bind(this);
+  }
+
+  handleFirstChange(event) {
```

Comment:
We need to find a better pattern for this.  One function per field is really ugly. Maybe we can use the "name" field of the element (which is stored in `event.target`) to figure out what state entry to store things in and then only have one function?